<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrjWizard</class>
 <widget class="QDialog" name="PrjWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Setup</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lPrjName">
     <property name="text">
      <string>Project name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="tbPrjName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lMapName">
     <property name="text">
      <string>Map name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="tbMapName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lLayers">
     <property name="text">
      <string>Layers:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="lbLayers">
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QGridLayout" name="layerLayout">
     <item row="3" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="lDimensions">
       <property name="text">
        <string>Layer dimensions [w, h]:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="sbHeight">
       <property name="suffix">
        <string> tiles</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sbWidth">
       <property name="suffix">
        <string> tiles</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="bUp">
       <property name="text">
        <string>Move up</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="bDown">
       <property name="text">
        <string>Move down</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="bDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="bAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="tbDescr">
       <property name="toolTip">
        <string>Layer description</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Layer description...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="controlLayout" stretch="0,0">
     <item>
      <widget class="QPushButton" name="bDone">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
